build/version.settings.targets

```diff
@@ -5,7 +5,7 @@
     <!-- Note: If you change the version number, make sure that you notify partner teams such as C++ IOT -->
     <MajorVersion>14</MajorVersion>
     <MinorVersion>0</MinorVersion>
-    <VersionZeroYear>2013</VersionZeroYear>
+    <VersionZeroYear>2019</VersionZeroYear>
```

Comment:
>2019 [](start = 21, length = 4)

Can you undo this commit once [https://github.com/microsoft/MIEngine/pull/951](https://github.com/microsoft/MIEngine/pull/951) is merged?

Reply:
Yes of course sorry this commit should even be in this PR. Should have been separate.